roles/ceph-client/tasks/pre_requisite.yml

```diff
@@ -24,8 +24,8 @@
   copy:
     src: "{{ fetch_directory }}/{{ fsid }}/etc/ceph/{{ cluster }}.client.admin.keyring"
     dest: "/etc/ceph/"
-    owner: "ceph"
-    group: "ceph"
+    owner: "{{ ceph_uid }}"
```

Comment:
This won't work since `ceph_uid` is only set for containerized deployment, this breaks non-containerized deployment gros, see: https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-defaults/tasks/facts.yml#L173